core/state_transition.go

```diff
@@ -197,7 +197,6 @@ func (st *StateTransition) TransitionDb() (ret []byte, usedGas uint64, failed bo
 	if err = st.useGas(gas); err != nil {
 		return nil, 0, false, err
 	}
-
```

Comment:
Again, this is a simple change, but can cause some extra work during merge with upstream. Or I'm missing something? (CC: @meowsbits)

Reply:
Seems to be fixed by @meowsbits